trestle/core/commands/validate.py

```diff
@@ -48,33 +45,10 @@ def _init_arguments(self) -> None:
             help=const.ARG_DESC_MODE + ' to validate.',
         )
 
-    def _run(self, args) -> None:
+    def _run(self, args: argparse.ArgumentParser) -> int:
         """Validate an OSCAL file in different modes."""
-        if args.file is None:
-            raise TrestleError(f'Argument "-{const.ARG_FILE_SHORT}" is required')
+        args_raw = args.__dict__
 
-        if args.mode is None:
-            raise TrestleError(f'Argument "-{const.ARG_MODE_SHORT}" is required')
-        mode = args.mode
-        if mode != const.VAL_MODE_DUPLICATES:
-            raise TrestleError(f'Mode value "{mode}" is not recognized.')
+        validator = validator_factory.create(args_raw[const.ARG_MODE])
```

Comment:
So one issue I have here is I have no way from the documentation you provide to discover what are valid modes?

The other question I have here is in the scenario you have provided there is an arbitrary list of arguments that can be provided and a user has no way to discover what they are. I'm presuming we will want to have some global arguments that are constant such as file and 

Finally I would think we would want to allow multiple modes to be passed in (via a comma separated list?).

Reply:
Again this is early and I only have one example validation mode.  It is now easier to add others.  I'm not even sure "file name" makes sense here.
The main thing is that I am assuming there will be different validation "modes" - and each will have a registered factory object.  And all the info needed to perform it will come from kwargs.